known-shortcuts/googlesheets.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
:zap: **Suggestion** 
The use of `as AppParams` for type assertion is acceptable, but it would be more idiomatic to define the type of the `app` object directly. This can help catch type errors earlier and improve code readability.